3rdParty/velocypack/src/Slice.cpp

```diff
@@ -345,6 +345,269 @@ unsigned int const SliceStaticData::FirstSubMap[32] = {
     0,  // 0x14, compact object, no index table - note: the offset is dynamic!
     0};
 
+uint64_t const SliceStaticData::PrecalculatedHashesForDefaultSeed[256] = {
+#ifdef VELOCYPACK_XXHASH
```

Comment:
What about a unit test for these values ?

Reply:
Tests added here:
https://github.com/arangodb/velocypack/commit/3748d5c49295e28f5dd8d7dfd0278ff8b645829b